ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -56,6 +57,7 @@ public class ThreeDSecureRequest implements Parcelable {
 
     private String nonce;
     private String amount;
+    private HashMap<String,Object> customFields;
```

Comment:
```suggestion
    private Map<String,Object> customFields;
```
Same best practice here for the internal variable declaration.